Application/Business/AttributeGroup/AttributeGroupServices.cs

```diff
@@ -22,34 +27,40 @@ public async Task<int> Count()
     {
         return await _attributeRepo.CountAsync();
     }
-    public async Task Create(AttributeGroupEntity attributeGroup)
+    public async Task Create(AttributeGroupEntity attributeGroup){
+        await _attributeRepo.CreateAsync(attributeGroup);
+    }
+    public async Task Update(Guid attributeGroupid, AttributeGroupEntity attributeGroup)
     {
-        var attList = List();
-        var check = false;
-        foreach (var attgrp in attList.Result)
+        var target = await GetById(attributeGroupid);
+        if(target is not null)
         {
-            if (attgrp.Name == attributeGroup.Name)
+            Type entityType = typeof(AttributeGroupEntity);
+            foreach (PropertyInfo propertyInfo in entityType.GetProperties())
```

Comment:
use Automapper handle this feature, prevent use the same logic in other places

Reply:
Updating in Controller and remove logic in service:
Controller: 
```
public async Task<IActionResult> UpdateAttributeGroup(Guid id, [FromBody] UpdateAttributeGroupRequest attributeGroup)
    {
        AttributeGroupEntity attGrpEnt = await _attributeServices.GetById(id);
        Mapper.Map(attributeGroup, attGrpEnt);
        await _attributeServices.Update(id, attGrpEnt);
        return AcceptedAtAction(nameof(GetAttributeGroup), new { id = id }, attGrpEnt);
    }
```
Service:
```
public async Task Update(Guid attributeGroupid, AttributeGroupEntity attributeGroup)
    {
        var target = await GetById(attributeGroupid);
        if(target is not null)
        {
            await _attributeRepo.UpdateAsync(target);
        }
        else
        {
            throw new NotFoundException("Attribute group not exist");
        }
    }
```